src/Forms/ValidatorList.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use InvalidArgumentException;
+use SilverStripe\ORM\ValidationResult;
+
+/**
+ * Class ValidatorList
+ *
+ * @package SilverStripe\Forms
+ */
+class ValidatorList extends Validator
```

Comment:
Not sure about the name ValidatorList. Does the order of the validators impact behaviour? If not, it's a set rather than a list. Aside from that my instinct would be to call this a CompositeValidator, as the name ValidatorList doesn't fully communicate that it is _itself_ a Validator.

Reply:
`CompositeValidator` it is! Thanks.